_includes/api/en/4x/req-is.md

```diff
@@ -9,13 +9,13 @@ Returns `false` otherwise.
 req.is('html');
 req.is('text/html');
 req.is('text/*');
-// => true
+// => 'html'
```

Comment:
Not all of these return `'html'`, only the first one does. Each of them return their input string.

Reply:
Ah, I didn't realize that (didn't look closely enough at `type-is`)